src/main/java/de/tum/in/www1/artemis/service/compass/CalculationEngine.java

```diff
@@ -18,21 +20,20 @@
     /**
      * Add a new assessment
      *
-     * @param assessment the new assessment as raw sting
-     * @param modelId the id of the corresponding model
+     * @param modelingAssessment the new assessment as raw sting
+     * @param submissionId       the id of the corresponding model
      */
-    void notifyNewAssessment(String assessment, long modelId);
+    void notifyNewAssessment(List<ModelElementAssessment> modelingAssessment, long submissionId);
```

Comment:
shouldn't we use plural, i.e. modelingAssessments? Can you also update the JavaDoc please as you do not handle a string any more?

Reply:
I wouldnt because it is a single assessment represented by a list of the individual model element assessments and treated as such in various other places.